addons/mail/static/src/models/guest.js

```diff
@@ -43,6 +50,13 @@ registerModel({
             required: true,
             readonly: true,
         }),
+        im_status: attr(),
+        /**
+         * States whether this partner is online.
```

Comment:
seems clear enough without the comments

Reply:
Copy/pasted from partner. The comment is also wrong...